tests/test_pytest/test_output_checkers.py

```diff
@@ -0,0 +1,53 @@
+# # # Distribution Statement A. Approved for public release. Distribution unlimited.
+# # #
+# # # Author:
+# # # Naval Research Laboratory, Marine Meteorology Division
+# # #
+# # # This program is free software: you can redistribute it and/or modify it under
+# # # the terms of the NRLMMD License included with this program. This program is
+# # # distributed WITHOUT ANY WARRANTY; without even the implied warranty of
+# # # MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the included license
+# # # for more details. If you did not receive the license, for more information see:
+# # # https://github.com/U-S-NRL-Marine-Meteorology-Division/
+
+"""Test all Output Checker plugins."""
+import pytest
+from PIL import Image
+import numpy as np
+
+from geoips.interfaces import output_checkers
+from geoips.commandline import log_setup
+
+
+log_setup.setup_logging()
+image = output_checkers.get_plugin("image")
+savedir = str(__file__).replace(
```

Comment:
Do we want to in general use $GEOIPS_OUTDIRS/scratch for these sorts of things?

Reply:
I was thinking that, however, I'm not sure if you can access environmental variables like that directly within python. 
Ie, I tested in IPython `print($GEOIPS_PACKAGES_DIR)`, which raised an error that it didn't exist.

Reply:
Generally, I think that should be fine. The data being written there, though, would it get cleaned up or would it stick around? Or would it get overwritten the next time that is called? I’m fine with that directory containing some stuff, but I’m concerned about it growing over time.

Reply:
It's overwritten each time. Max of 27 images in that directory at all times.

Reply:
Okay, yeah, that sounds fine to me. 

Thinking about this, do you see any reason to retain the files for tests that pass? Maybe it would be a good idea to clean up the scratch files for any tests that pass?

Reply:
I do see a reason. This way, we can visually check that each file passes or fails in the way we expect. For other output checkers I would be fine removing these files, however for images specifically, I see the need to keep them.

Reply:
Okay, keeping them is fine by me so long as they're being overwritten. I just want to be sure that directory won't grow over time.

Reply:
Unless we end up changing the filenames dynamically, it will never grow. Going to resolve this for now.